### PR TITLE
Dashboard integration

### LINKED ISSUE #18: IN: Dashboard, polls, friend lists
- On the dashboard, I can see my friends lists and polls (fetched from the backend) and create new polls/friend lists
- https://github.com/hatchways/team-cheerios/blob/dev/server/controllers/pollControllers.js we need a /fetchPolls controller that fetches multiple polls
- https://github.com/hatchways/team-cheerios/blob/dev/server/controllers/friendsListControllers.js modify this controller that only fetches friend lists that the user id created

### PR DESCRIPTION
**Pull request by aalthaf**
- Can see friendsList and polls fetched from backend

- add poll controllers

### CODE REVIEW

File: client/src/components/FriendsListDialog/FriendsListDialog.js [hunk 1]
```diff
@@ -71,10 +71,19 @@ export default function FriendsListDialog({ open, onClose, ...rest }) {
...
-  // TODO: fetch friends
-  const friends = [];
-  const loading = true;
+  const [friends, setFriends] = React.useState([]);
+
+  // TODO: fetch friends user user ids
+  React.useEffect(() => {
+    if (open) {
+      axios
+        .get("/friends")
```
Comment (yoniakabecky) [inline client/src/components/FriendsListDialog/FriendsListDialog.js]: `/friends` return friend suggestions.
`/friends/followings` will return friends you're following

We need to fetch "followings" here I think 🤔

File: client/src/components/ProfileView/ProfileView.js [hunk 2]
```diff
@@ -45,13 +45,33 @@ export default function ProfileView() {
   const classes = useStyles();
   const [openFriendDialog, setOpenFriendDialog] = React.useState(false);
   const [openPollDialog, setOpenPollDialog] = React.useState(false);
+  const [myFriendsLists, setMyFriendsLists] = React.useState([]);
+  const [myPolls, setMyPolls] = React.useState([]);
+
+  React.useEffect(() => {
+      const requestOne = axios.get("/friends-list");
+      const requestTwo = axios.get("/poll");
```
Comment (sina-jamshidi) [inline client/src/components/ProfileView/ProfileView.js]: just best practice-wise, you're going to want to name these something better. There's nothing wrong with longer names: `friendsListRequest`, `pollRequest`

**Review by sina-jamshidi — APPROVED** [review]
Looks good to me!

File: client/src/components/ProfileView/ProfileView.js (hunk 2 above)
Comment (aalthaf) [reply client/src/components/ProfileView/ProfileView.js]: removed this as one rejection causes issues. For ex: no polls and some friendslist

**Pull request merged**